browser/js/modalWindow/modal.factory.js

```diff
@@ -0,0 +1,10 @@
+app.factory('ModalFactory', function($http){
```

Comment:
I think its best to keep all factory methods for a specific route in 1 factory -- easier long-term code maintenance if you know where all "kit route methods" are.  In this case, maybe you can move this "postKit" method into the KitsFactory. => browser/js/kit/kit.factory.js